src/Components/Nav/Nav.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from "react";
+import { API_Detail } from "../../config";
+import "./Nav.scss";
+import NavDropdown from "./NavDropdown";
+
+export default class Nav extends Component {
+  state = {
+    menus: [],
+    activeId: false,
+    isLogin: false,
+    click: false
+  };
+
+  componentDidMount() {
+    if (localStorage.getItem("authorization")) {
+      this.setState({ isLogin: true });
+    }
```

Comment:
componentDidMount에서 setState를 하게 되면 render 되자마자 또 render가 되서 불필요하게 두 번 render가 됩니다!
그래서 다음과 같이 state에서 바로 값을 넣어주시면 됩니다. 
```suggestion
  state = {
    menus: [],
    activeId: false,
    isLogin: localStorage.getItem("authorization") ? true : false,
    click: false
  };

  componentDidMount() {
```

또한 여기서 token값이 존재한다는 것 자체가 true false로 나올 수 있어서 아래처럼 한 번 더 줄일 수 있습니다.
```suggestion
  state = {
    menus: [],
    activeId: false,
    isLogin: localStorage.getItem("authorization"),
    click: false
  };
```

Reply:
오호!!😎😎